### ISSUE TITLE
How to create custom element for ContentEdit

### ISSUE THREAD

**Issue by Eeliya** [GetmeUK/ContentTools#147]
I want to have a class `ContentEdit.MyCustomP` that handles all the `<p class='.my-custom-p'></p>` or `<p data-my-custom-p></p>` and all the other `p` tags get handled by the `ContentEdit.Text`

**Comment by anthonyjb** [comment]
Hi @Eeliya , You can override the Element class used to parse an element using the `data-ce-tag`  attribute. Please read the documentation on custom tag names here: http://getcontenttools.com/api/content-edit#tag-names

**Comment by Eeliya** [comment]
Thank you,

it would be great if you could also add an example element to that tutorial :)

**Comment by anthonyjb** [comment]
Yes we're short off an example on how to build custom tags and I agree we should have one.
